app/controllers/tasks_controller.rb

```diff
@@ -78,19 +74,27 @@ def destroy
     end
   end
 
-  def confirm 
-    　@task = current_user.tasks.build(task_params)
-    　 render :new if @task.invalid? 
-  end    
+  def confirm
+    @task = current_user.tasks.build(task_params)
+    render :new if @task.invalid?
+  end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_task
-      @task = Task.find(params[:id])
-    end
 
-    # Only allow a list of trusted parameters through.
-    def task_params
-      params.require(:task).permit(:name, :content, :end_date, :priority, :status)
+  # Use callbacks to share common setup or constraints between actions.
+  def set_task
+    @task = Task.find(params[:id])
+  end
+
+  def task_params
+    params.require(:task).permit(:name, :content, :end_date, :priority, :status)
+  end
+
+  def login_check
+    if logged_in? 
+      return true
+    else
+      redirect_to new_session_path, notice: t('view.task.login')
     end
```

Comment:
`return true`させる必要ありますか？返り値のtrueをどこで使っているかを意識して欲しいです。
```suggestion
      redirect_to new_session_path, notice: t('view.task.login') unless logged_in? 

```

Reply:
勉強になりました。ありがとうございます。

6c0a391